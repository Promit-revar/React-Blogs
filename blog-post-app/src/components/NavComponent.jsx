import React from 'react';
import '../assets/style.css';
export default function NavComponent() {
    return (
        <nav >
        <div className='logo'>
            <h1>The Artifact</h1>
            <em>Culture &amp; Art blog</em>
        </div>
        <div className='nav-links'>
            <a to='#'>Home</a>
            <a to='#'>About</a>
            <a to='#'>Contact</a>
        </div>
        </nav>
        
    );  
}
