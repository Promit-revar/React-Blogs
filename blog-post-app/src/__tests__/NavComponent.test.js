import NavComponent from '../components/NavComponent';
import { render } from '@testing-library/react';
import React from 'react';
describe('Nav Component Snapshot', () => {
  it('should render the Nav component', () => {
    const {asFragment} = render(<NavComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
