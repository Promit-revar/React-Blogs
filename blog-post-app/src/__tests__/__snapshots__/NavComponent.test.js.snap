// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component Snapshot should render the Nav component 1`] = `
<DocumentFragment>
  <nav>
    <div
      class="logo"
    >
      <h1>
        The Artifact
      </h1>
      <em>
        Culture & Art blog
      </em>
    </div>
    <div
      class="nav-links"
    >
      <a
        to="#"
      >
        Home
      </a>
      <a
        to="#"
      >
        About
      </a>
      <a
        to="#"
      >
        Contact
      </a>
    </div>
  </nav>
</DocumentFragment>
`;
